src/TLuaInterpreter.cpp

```diff
@@ -11361,6 +11895,26 @@ bool TLuaInterpreter::validLuaCode(const QString &code)
     return error == 0;
 }
 
+std::pair<bool, QString> TLuaInterpreter::discordApiEnabled(lua_State* L, bool writeAcces)
+{
+    mudlet* pMudlet = mudlet::self();
+
+    if (!pMudlet->mDiscord.libraryLoaded()) {
+        return make_pair(false, QStringLiteral("Discord API is not available"));
+    }
+
+    auto& host = getHostFromLua(L);
+    if (!(host.mDiscordAccessFlags & Host::DiscordLuaAccessEnabled)) {
+        return make_pair(false, QStringLiteral("Discord API is disabled in settings for privacy"));
+    }
+
+    if (writeAcces && !pMudlet->mDiscord.discordUserIdMatch(&host)) {
+        return make_pair(false, "Discord API is read-only as you're logged in with a different account in Discord compared to the one you entered for this profile");
```

Comment:
Aren't you missing a `StringLiteral` here?

Reply:
Thanks, fixed